python/pyspark/sql/connect/readwriter.py

```diff
@@ -514,6 +586,7 @@ def _test() -> None:
         del pyspark.sql.connect.readwriter.DataFrameReader.load.__doc__
         del pyspark.sql.connect.readwriter.DataFrameReader.option.__doc__
         del pyspark.sql.connect.readwriter.DataFrameReader.text.__doc__
+        del pyspark.sql.connect.readwriter.DataFrameReader.csv.__doc__
```

Comment:
Hm, does the test fail?

Reply:
It passed locally, was working against an older master branch.